paimon-web-ui/src/views/metadata/components/table-column-content/index.tsx

```diff
@@ -26,10 +26,12 @@ const props = {
     type: Array as PropType<ColumnDTO[]>,
     default: () => [],
   },
+  'isEdit': Boolean,
```

Comment:
Need to use proptype of vue for packaging

Reply:
done.